tests/DependencyInjection/NeustaConverterExtensionTest.php

```diff
@@ -123,6 +126,122 @@ public function test_with_mapped_context(): void
         self::assertSame('age', $ageInYearsPopulator->getArgument('$contextProperty'));
     }
 
+    public function test_with_converting_populator(): void
+    {
+        $container = $this->buildContainer([
+            'populators' => [
+                'foobar' => [
+                    'converter' => GenericConverter::class,
+                    'property' => [
+                        'targetTest' => 'sourceTest',
+                    ],
+                ],
+            ],
+        ]);
+
+        // converter
+        $populator = $container->getDefinition('foobar');
+
+        self::assertSame(ConvertingPopulator::class, $populator->getClass());
+        self::assertTrue($populator->isPublic());
+        self::assertInstanceOf(TypedReference::class, $populator->getArgument('$converter'));
+        self::assertSame('targetTest', $populator->getArgument('$targetPropertyName'));
+        self::assertSame('sourceTest', $populator->getArgument('$sourcePropertyName'));
+    }
+
+    public function test_with_array_converting_populator(): void
+    {
+        $container = $this->buildContainer([
+            'populators' => [
+                'foobar' => [
+                    'converter' => GenericConverter::class,
+                    'property' => [
+                        'targetTest' => 'sourceTest',
+                    ],
+                ],
+            ],
+        ]);
+
+        // converter
+        $populator = $container->getDefinition('foobar');
+
+        self::assertSame(ConvertingPopulator::class, $populator->getClass());
+        self::assertTrue($populator->isPublic());
+        self::assertInstanceOf(TypedReference::class, $populator->getArgument('$converter'));
+        self::assertSame('targetTest', $populator->getArgument('$targetPropertyName'));
+        self::assertSame('sourceTest', $populator->getArgument('$sourcePropertyName'));
+    }
```

Comment:
This doesn't test the `ArrayConvertingPopulator`. It's exactly the same as the one above it.